20200911_Person/src/de/telran/data/Person.java

```diff
@@ -0,0 +1,70 @@
+package de.telran.data;
+
+import java.util.Arrays;
+import java.util.Comparator;
+
+public class Person {
+    private String name;
+    private int age;
+
+    //Конструктор
+    public Person(String name, int age) {
+        this.name = name;
+        this.age = age;
+    }
+
+    public String getName() {
+        return name;
+    }
+
+    public void setName(String name) {
+        this.name = name;
+    }
+
+    public int getAge() {
+        return age;
+    }
+
+    public void setAge(int age) {
+        //В этом методе можем сделать защиту от неправильных данных
+        if (age > 0 && age < 110) {
+            this.age = age;
+        } else {
+            System.out.println("Wrong age!");
+        }
+    }
+
+    @Override
+    public String toString() {
+        return "Name: " + name +
+                ", age " + age + "\n";
+    }
+
+    public static void outPrint(Person[] array) {
+        for (Person i : array) {
+            System.out.print(i);
+        }
+        System.out.println();
+    }
+
+    //public - так как будем использовать метод не внутри класса
+    //static - так как будем перебирать группу людей
+    public static void findOldestPerson(Person[] array) {
+        Arrays.sort(array, Comparator.comparing(Person::getAge));
```

Comment:
It is better not to use Collections yet when solving tasks. Sorting in the solution is redundant. This is an "expensive" operation. It is possible to solve a task without using sorting.